Library/ViewModels/CommentComposerViewModel.swift

```diff
@@ -95,6 +103,12 @@ public final class CommentComposerViewModel:
     self.postButtonPressedProperty.value = ()
   }
 
+  fileprivate let resetInputProperty = MutableProperty(())
+  public func resetInput() {
+    self.bodyTextDidChange(nil)
```

Comment:
Let's not call this other input from this input, instead in our `init` we should wire them up like:

```swift
self.bodyText = Signal.merge(
  self.bodyTextDidChangeProperty.signal,
  self.resetInputProperty.signal.mapConst(nil)
)
```